samples/net/wpanusb/README.rst

```diff
@@ -31,22 +33,21 @@ There are configuration files for various setups in the
 - :file:`overlay-cc2520.conf`
     This overlay config enables support for CC2520
 
-Build the WPANUSB sample like this:
+Build the WPANUSB sample for a board:
```

Comment:
You changed ``WPANUSB`` to ``wpanusb`` on line 20.  What's the correct spelling?

Reply:
Changed to wpanusb